<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageBook</class>
 <widget class="QDialog" name="ManageBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>357</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Manage Book</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_isbn">
            <property name="text">
             <string>ISBN:</string>
            </property>
            <property name="buddy">
             <cstring>line_author</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_isbn"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_title">
            <property name="text">
             <string>Title:</string>
            </property>
            <property name="buddy">
             <cstring>line_author</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_title"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_description">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_description"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_author">
            <property name="text">
             <string>Author ID:</string>
            </property>
            <property name="buddy">
             <cstring>line_author</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_author"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_cover">
            <property name="text">
             <string>Cover Image URL:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_cover"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_genre">
            <property name="text">
             <string>Genre:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_genre"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_quantity">
            <property name="text">
             <string>Quantity:</string>
            </property>
            <property name="buddy">
             <cstring>line_author</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_quantity"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
